### ISSUE TITLE
what is the meaning if the code you added in line 103?

### ISSUE THREAD

**Issue by zhujinwen0924** [threeYANG/libtorch-SSD#7]
elif self.phase == "trace":
        output = torch.cat((loc.view(loc.size(0), -1, 4), self.softmax(conf.view(conf.size(0), -1, self.num_classes))), 2)
hello,can you tell me why you add this code?thanks very much.
